lib/builder.js

```diff
@@ -138,16 +138,16 @@ YUI.add('doc-builder', function (Y) {
          * Wrapper around the Markdown parser so it can be normalized or even side stepped
          * @method markdown
          * @private
-         * @param {String} md The Markdown string to parse
+         * @param {String} data The Markdown string to parse
          * @return {HTML} The rendered HTML
          */
-        markdown: function (md) {
-            var html = marked(md, this.options.markdown);
+        markdown: function (data) {
+            var md = new MarkdownIt(this.options.markdown);
```

Comment:
Can we create the instance only once? Also, what are the defaults for `this.options.markdown`?